merkle/objhasher/objhasher.go

```diff
@@ -0,0 +1,51 @@
+// Copyright 2016 Google Inc. All Rights Reserved.
```

Comment:
It should be the year of creation. So yes, that's 2017
Follows the discussion in #312.